src/components/search.js

```diff
@@ -27,16 +40,35 @@ export default class Search extends React.PureComponent {
     )
   }
 
+  handleClick() {
+    if (this.input.value == '') return
+    this.clear()
+
+    this.props.onSearch(
+      this.emojiIndex.search('', {
+        emojisToShowFilter: this.props.emojisToShowFilter,
+        maxResults: this.props.maxResults,
+        include: this.props.include,
+        exclude: this.props.exclude,
+        custom: this.props.custom,
+      }),
+    )
```

Comment:
This is duplicated pretty much exactly in `handleChange`. Let's pull this out into a new function on the class. Something like: 
```js
search(value) {
    this.props.onSearch(
      this.emojiIndex.search(value, {
        emojisToShowFilter: this.props.emojisToShowFilter,
        maxResults: this.props.maxResults,
        include: this.props.include,
        exclude: this.props.exclude,
        custom: this.props.custom,
      }),
    )
  }
```
and then just call `this.search('')` here and `this.search(value)` in `handleChange`.